indexing-service/src/main/java/org/apache/druid/indexing/input/DruidSegmentReader.java

```diff
@@ -270,6 +273,17 @@ public boolean hasNext()
           theEvent.put(metric, value);
         }
       }
+
+      /*
```

Comment:
Could you change this to single-line comments? We don't usually use multi-line comment.

Reply:
Done